ext/IntegralsFastGaussQuadratureExt.jl

```diff
@@ -0,0 +1,52 @@
+module IntegralsFastGaussQuadratureExt
+using Integrals
+if isdefined(Base, :get_extension)
+    import FastGaussQuadrature
+    import FastGaussQuadrature: gausslegendre
+    # and eventually gausschebyshev, etc.
+else
+    import ..FastGaussQuadrature
+    import ..FastGaussQuadrature: gausslegendre
+end
+using LinearAlgebra
+
+Integrals.gausslegendre(n) = FastGaussQuadrature.gausslegendre(n)
+
+function gauss_legendre(f, p, lb, ub, nodes, weights)
+    scale = (ub - lb) / 2
+    shift = (lb + ub) / 2
+    scaled_f = s -> f(scale * s + shift, p)
```

Comment:
We don't need to define the closure here.

Reply:
I've changed it now - is this what you mean to have instead?